src/tx/action.service.ts

```diff
@@ -39,8 +40,9 @@ export class ActionService {
               this.encodeFungibleAssetValue({
                 currency: CURRENCIES[job.ticker],
                 rawValue: BigInt(
-                  job.amount *
-                    Math.pow(10, CURRENCIES[job.ticker].decimalPlaces),
+                  new Decimal(job.amount)
+                    .times(Math.pow(10, CURRENCIES[job.ticker].decimalPlaces))
+                    .toString(),
```

Comment:
I think it should use `Decimal#pow` because `10**18` is bigger than `Number.MAX_SAFE_INTEGER`.

https://mikemcl.github.io/decimal.js/#Dpow

```suggestion
                  new Decimal(job.amount)
                    .times(new Decimal(10).pow(CURRENCIES[job.ticker].decimalPlaces))
                    .toString(),
```

Also, I confused about what `.times`' function, it was the same as `.mul`.